app/serializers/event.js

```diff
@@ -2,13 +2,14 @@ import ApplicationSerializer from 'open-event-frontend/serializers/application';
 import CustomPrimaryKeyMixin from 'open-event-frontend/mixins/custom-primary-key';
 
 export default ApplicationSerializer.extend(CustomPrimaryKeyMixin, {
-  primaryKey: 'attributes.identifier',
 
-  attrs: {
+  primaryKey : 'attributes.identifier',
+  attrs      : {
     type             : 'event-type',
     topic            : 'event-topic',
     subTopic         : 'event-sub-topic',
     copyright        : 'event-copyright',
-    externalEventUrl : 'event-url'
+    externalEventUrl : 'event-url',
+    speakersCall     : 'speakers-call'
```

Comment:
Also @CosmicCoder96 could you also remove `externalEventUrl : 'event-url',` not needed anymore. The API response attribute name has changed.